packages/victory-group/src/victory-group.tsx

```diff
@@ -178,19 +199,22 @@ VictoryGroup.defaultProps = {
   theme: VictoryTheme.grayscale,
 };
 
-// We need to attatch the static properties to the memoized version, or else
+// TODO: This utility could be moved and used for other components
+const typedMemo: <T>(component: T, equalityCheck?: any) => T = React.memo;
```

Comment:
`Memo`izing doesn't play well with TypeScript, so this utility (thanks to others who suggested the idea) could help with getting some of the other components across the board, allowing a memoized component to be typed appropriately.